### ISSUE TITLE
Define IDL value to attributes for using JSX etc.

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#525]
```jsx
<Component contentEditable="inherit" />
```

`inherit` is the IDL value (that is state). But it is disallowed as an HTML native attribute value.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/084f03bfe7bac41da8ac10f46988e454c219ad33/types/react/v17/index.d.ts#L1817-L1833

**Comment by YusukeHirao** [comment]
- [ ] Need IDL data for schemas.

**Comment by YusukeHirao** [comment]
I may add IDL values to schemas:

https://github.com/markuplint/markuplint/blob/923dbcb2f81cdc7702e1e73426e2455daf8b20ac/packages/%40markuplint/html-spec/src/spec-common.attributes.json#L30-L41

```diff
 "contenteditable": { 
 	"type": { 
 		"enum": ["", "true", "false"], 
++ 		"idlValues": ["inherit"],
 		"disallowToSurroundBySpaces": true, 
 		"invalidValueDefault": "inherit", 
 		"missingValueDefault": "inherit", 
 		"sameStates": { 
 			"true": [""] 
 		}
 	} 
 }, 
```
